src/shared/components/Banners/SaveSuccessBanner/SaveSuccessBanner.tsx

```diff
@@ -0,0 +1,13 @@
+import { useTranslation } from 'react-i18next';
+
+import { Banner, BannerProps } from '../Banner';
+
+export const SaveSuccessBanner = ({ children, ...props }: BannerProps) => {
+  const { t } = useTranslation('app');
+
+  return (
+    <Banner duration={3500} severity="success" data-testid="save-success-banner" {...props}>
```

Comment:
Could you please create a variable for the duration and place it in the SaveSuccessBanner.const.ts file?